mmil.raw.html

```diff
@@ -6295,11 +6295,10 @@
 
 <TR>
   <TD>seqcl</TD>
-  <TD>~ iseqcl</TD>
-  <TD>iseqcl requires that ` F ` be defined on ` ( ZZ>= `` M ) `
-  not merely ` ( M ... N ) ` as in seqcl . This is not a problem
-  when used on infinite sequences, but perhaps this requirement
-  could be relaxed if there is a need.</TD>
+  <TD>~ iseqcl , ~ iseqclt</TD>
+  <TD>~ iseqcl requires that ` F ` be defined on ` ( ZZ>= `` M ) `
+  not merely ` ( M ... N ) ` as in seqcl . This requirement is
```

Comment:
I guess there should be no space in `seqcl.` ? When you don't put tildes it's because it's a result of set.mm, is that it ? (same with `fsumabs )`). One way to disambiguate would be to write, e.g., `set.mm/seqcl`.

Reply:
The layout of the table is mainly what says what is from set.mm and what is from iset.mm (left column set.mm, middle column iset.mm).

I guess I'll go with just removing "as in seqcl" as I'm not sure much is gained by repeating the name of the theorem.